citesphere/src/main/webapp/WEB-INF/views/auth/group/editItem.jsp

```diff
@@ -141,11 +145,26 @@ $(function() {
 		var authId = $(this).attr('data-authority-id');
 		$("#uriEditorLocalId").val(authId);
 		$("#editorAuthorityUsed").html("Using stored authority entry <i>" + $(this).attr('data-authority-name') + "</i>.");
+		showPersonNameInModal($(this).attr('data-authority-name'), "Editor");
 		$("#uriLoadingFoundEditor").popover('hide');
 		event.preventDefault();
 	});
 });
 
+function showPersonNameInModal(name, personType){
+	// To remove the year from name
+	if((name.split(",").length-1) == 2) {
```

Comment:
why don't you check if the length equals 3?

Reply:
Also, I think I would check for > 2. I think, we can assume that names won't have commas in them, so that it will have the format Last, First, something else.